tools/extras/prebuild.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
This needs to be a small Python3 script, not bash, to support Windows.  We include Python in our toolchain distribution on all hosts just for such small scripting needs.